docs/packages/pkg/Fruit.rst

```diff
@@ -0,0 +1,43 @@
+.. spelling::
+
+    Google Fruit
+
+.. index:: testing ; Fruit
+
+.. _pkg.Fruit:
+
+Fruit
+=====
+
+-  `Official <https://github.com/google/fruit>`__
+-  `Hunterized <https://github.com/hunter-packages/fruit>`__
+-  `Dev branch <https://github.com/ruslo/hunter/tree/Fruit>`__
+-  `Versions <https://github.com/ruslo/hunter/blob/master/cmake/projects/Fruit/hunter.cmake>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/Fruit/CMakeLists.txt>`__
+-  `Added by Alexey Shevchenko <https://github.com/FelikZ>`__
+   (`pr-1527 <https://github.com/ruslo/hunter/pull/1527>`__)
+
+.. code-block:: cmake
+
+    hunter_add_package(Fruit)
```

Comment:
Use `.. literalinclude::` to include code between `# DOCUMENTATION_START {` and `# DOCUMENTATION_END }` from CMakeLists.txt of example, see:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation
* https://raw.githubusercontent.com/ruslo/hunter/master/docs/packages/pkg/foo.rst